vrpn_3DConnexion.C

```diff
@@ -24,12 +24,14 @@ typedef struct input_devinfo {
 // USB vendor and product IDs for the models we support
 static const vrpn_uint16 vrpn_3DCONNEXION_VENDOR = 0x046d; //1133;	// 3Dconnexion is made by Logitech
 static const vrpn_uint16 vrpn_SPACEMOUSEWIRELESS_VENDOR = 9583; 	// Made by a different vendor...
+static const vrpn_uint16 vrpn_SPACEMOUSECOMPACT_VENDOR = vrpn_SPACEMOUSEWIRELESS_VENDOR;
```

Comment:
This seems like an extra level of indirection that we don't need.

Reply:
Happy to change it, but I thought it would make it more clear that the vendor is the same for these two devices rather than duplicating the id number.

Reply:
Why not just re-use the vrpn_SPACEMOUSEWIRELESS_VENDOR in the Compact device descriptor?

Reply:
I suspect it would look like copy pasta to a reader of the code, even though it is correct. I'll change it, though.

Reply:
I agree that we don't want static const vrpn_uint16 vrpn_SPACEMOUSECOMPACT_VENDOR = 9583;

I'm suggesting removing vrpn_SPACEMOUSECOMPACT_VENDOR from the code altogether.  There are a bunch of devices that all use vrpn_3DCONNEXION_VENDOR for example, rather than having a different vendor for each.  The vrpn_SPACEMOUSEWIRELESS_VENDOR is probably the wrong name to use; maybe vrpn_3DCONNEXIONWIRELESS_VENDOR?  I think they may have gotten bought in the meantime...

Reply:
We don't need to change it.